sw/device/tests/BUILD

```diff
@@ -3439,6 +3442,7 @@ opentitan_test(
         EARLGREY_TEST_ENVS,
         EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
     ),
+    verilator = verilator_params(tags = ["manual"]),
```

Comment:
Why manual? Isn't this test that needs a spi flash on the bus and therefore can't be tests in verilator?